gallery/src/components/NxAccordion/NxAccordionPage.tsx

```diff
@@ -160,6 +162,14 @@ const NxAccordionPage = () =>
       A simple example of an <NxCode>NxAccordion</NxCode>.
     </GalleryExampleTile>
 
+    <GalleryExampleTile title="Example with nested NxList"
+                        id="nx-accordion-nested-nx-list-example"
+                        defaultCheckeredBackground={true}
+                        liveExample={NxAccordionWithNxListExample}
+                        codeExamples={NxAccordionWithNxListCode}>
+      An example of an <NxCode>NxAccordion</NxCode> with nested <NxCode>NxList</NxCode>.
```

Comment:
I think it would be worthwhile to add a brief mention of this be a special stylistic case which is why this example exists

Reply:
> I think it would be worthwhile to add a brief mention of this be a special stylistic case which is why this example exists

Good point! I added an extra description here: dad83b7bca34e011e27cd8acb6202874e227dd82